content/intro-to-storybook/react/ko/data.md

```diff
@@ -2,151 +2,165 @@
-    case actions.PIN_TASK:
-      return taskStateReducer('TASK_PINNED')(state, action);
-    default:
-      return state;
-  }
-};
-
-// The initial state of our store when the app loads.
-// Usually you would fetch this from a server
 const defaultTasks = [
   { id: '1', title: 'Something', state: 'TASK_INBOX' },
   { id: '2', title: 'Something more', state: 'TASK_INBOX' },
   { id: '3', title: 'Something else', state: 'TASK_INBOX' },
   { id: '4', title: 'Something again', state: 'TASK_INBOX' },
 ];
 
-// We export the constructed redux store
-export default createStore(reducer, { tasks: defaultTasks });
+/*
+ * 저장소는 여기서 만들어집니다.
+ * 'slice'의 자세한 정보는 아래 문서에서 확인할 수 있습니다.
+ * https://redux-toolkit.js.org/api/createSlice
+ */
+const TasksSlice = createSlice({
+  name: 'tasks',
+  initialState: defaultTasks,
+  reducers: {
+    updateTaskState: (state, action) => {
+      const { id, newTaskState } = action.payload;
+      const task = state.findIndex(task => task.id === id);
+      if (task >= 0) {
+        state[task].state = newTaskState;
+      }
+    },
+  },
+});
+
+// slice 속 포함된 액션 사용을 위해 컴포넌트로부터 내보내집니다. 
+export const { updateTaskState } = TasksSlice.actions;
+
+/*
+ * 앱의 저장소 환경설정은 다음과 같습니다.
+ * 리덕스의 configureStore 의 자세한 정보는 아래 문서에서 확인할 수 있습니다.
+ * https://redux-toolkit.js.org/api/configureStore
+ */
+const store = configureStore({
+  reducer: {
+    tasks: TasksSlice.reducer,
+  },
+});
+
+export default store;
 ```
 
-그런 다음 `TaskList` 컴포넌트에서 default export를 업데이트하여 redux store에 연결하고 task를 렌더링 합니다.
-
-```javascript
-// src/components/TaskList.js
+다음 `TaskList` 컴포넌트를 Redux store와 연결하고, 알고자 하는 task들을 렌더링 하기 위해 업데이트합니다.
 
+```js:title=src/components/TaskList.js
 import React from 'react';
 import PropTypes from 'prop-types';
 
 import Task from './Task';
-import { connect } from 'react-redux';
-import { archiveTask, pinTask } from '../lib/redux';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { updateTaskState } from '../lib/store';
 
 export function PureTaskList({ loading, tasks, onPinTask, onArchiveTask }) {
-  /* previous implementation of TaskList */
+  /* TaskList의 이전 구현 */
 }
 
 PureTaskList.propTypes = {
-  /** Checks if it's in loading state */
+  /** loading 상태인지 확인하는 데이터 */
   loading: PropTypes.bool,
-  /** The list of tasks */
+  /** 작업 목록 데이터 */
   tasks: PropTypes.arrayOf(Task.propTypes.task).isRequired,
-  /** Event to change the task to pinned */
+  /** 작업을 고정으로 변경하는 이벤트 */
   onPinTask: PropTypes.func.isRequired,
-  /** Event to change the task to archived */
+  /** 작업 아카이빙을 위한 이벤트 */
   onArchiveTask: PropTypes.func.isRequired,
 };
 
 PureTaskList.defaultProps = {
   loading: false,
 };
 
-export default connect(
-  ({ tasks }) => ({
-    tasks: tasks.filter((t) => t.state === 'TASK_INBOX' || t.state === 'TASK_PINNED'),
-  }),
-  (dispatch) => ({
-    onArchiveTask: (id) => dispatch(archiveTask(id)),
-    onPinTask: (id) => dispatch(pinTask(id)),
-  })
-)(PureTaskList);
+export function TaskList() {
```

Comment:
@lisohh I was looking at the pr and it seems that your fork is not up to date with the latest version. As the translation is still mentioning the presentational/container pattern. We've updated this version of the tutorial with a recent pull request, if you could pull in the latest changes and adjust accordingly I'd appreciate it. As it would improve the translation tremendously.

Reply:
@jonniebigodes jonniebigodes we've cheked all the feedbacks and updated our translation.
Please check again. Thank you for waiting.  :)